README.md

```diff
@@ -188,6 +188,10 @@ The server provides the following tools:
   - Parameters:
     - `collection_name`: Name of collection to release
 
+- `milvus_get_collection_info`: List the statistics collected on a specific collection
+  - Parameters:
+    - `collection_name`: Name of collection to release
```

Comment:
Instead of list statistics, the method lists the collection details like schema, properties, and time created.
Also, the explanation of `collection_name` is incorrect as well - this method does not release collection.